packages/attribute-slicer/src/selection/SelectionManager.ts

```diff
@@ -248,6 +248,9 @@ export default class SelectionManager<T extends ISelectableItem<any>> {
         if (this._dragging) {
             this._dragging = false;
             if (this.brushMode) {
+                if (this.keyState.ctrl) {
+                    this._brushingSelection.push.apply(this._brushingSelection, this.selection);
```

Comment:
Why not just `this._brushingSelection.push(this.selection`? Does `fn.apply` resolve some kind of `this` issue?